gradunwarp/core/unwarp_resample.py

```diff
@@ -268,36 +245,26 @@ def non_linear_unwarp_siemens(self, volshape, dv, dxyz, m_rcs2lai, m_rcs2lai_noh
 
 
             #im_ = utils.interp3(self.vol, vrcsw.x, vrcsw.y, vrcsw.z)
-            ndimage.interpolation.map_coordinates(self.vol,
-                                                  vrcsw,
-                                                  output=im_,
-                                                  order=self.order)
+            ndimage.map_coordinates(self.vol,
+                                    vrcsw,
+                                    output=im_,
+                                    order=self.order)
             # find NaN voxels, and set them to 0
-            im_[np.where(np.isnan(im_))] = 0.
-            im_[np.where(np.isinf(im_))] = 0.
+            np.nan_to_num(im_, copy=False, posinf=0, neginf=0)
```

Comment:
There are python2 tests, so I assume it was working previously.  Looks like the python2 version of numpy doesn't know about `posinf` here?

Reply:
Just for reference, the last version of numpy to support Python 2 was [1.16](https://numpy.org/doc/1.16/).

Reply:
Maybe we should just drop support of Python 2?

Reply:
It looks like he fixed it, so I don't see a benefit to dropping python2 immediately.